### DISCUSSION TITLE
notebook/matrix_square_root

### DISCUSSION THREAD

**Discussion by giscus** [danmackinlay/danmackinlay.name-comments#5] [Comments]
# notebook/matrix_square_root

Whitening, preconditioning etc

https://danmackinlay.name/notebook/matrix_square_root.html

**Comment by BenOwell** [comment]
Hi Dan, Thanks for drawing attention to this paper and the generalized Woodbury formula. I wouldn't have guessed it would work... but it does. I can see it being very useful indeed.

**Comment by danmackinlay** [comment]
In act, I'm grateful to you @BenOwell I had completely forgotten this algorithm since I wrote it up, and it may have been useful on my last project had I remembered.
